AU2/plugins/custom_plugins/WantedPlugin.py

```diff
@@ -102,130 +117,122 @@ def on_event_update(self, e: Event, htmlResponse) -> List[HTMLComponent]:
-                player = ASSASSINS_DATABASE.get(playerID)
-                if player.is_police:
-                    wanted_police[playerID] = (duration, crime, redemption)
-                else:
-                    wanted_players[playerID] = (duration, crime, redemption)
-
-            # check to see if a victim died while wanted
-            for (killer, victim) in e.kills:
-                if victim in most_recent_wanted:
-                    wanted_expiry, wanted_data = most_recent_wanted[victim]
-                    if wanted_expiry < e.datetime:
-                        continue
-                    victim_assassin = ASSASSINS_DATABASE.get(victim)
-                    if victim_assassin.is_police:
-                        dead_police[victim] = wanted_data
-                        silentpop(wanted_police, victim)
-                    else:
-                        dead_players[victim] = wanted_data
-                        silentpop(wanted_players, victim)
-
-        """
-        
-    PLAYER_TABLE_ROW_TEMPLATE = 
-    "<tr><td>{REAL NAME}</td><td>{PSEUDONYMS}</td><td>{ADDRESS}</td><td>{COLLEGE}</td><td>{WATER_STATUS}</td><td>{CRIME}</td><td>{REDEMPTION}</td><td>{NOTES}</td></tr>"
-    POLICE_TABLE_ROW_TEMPLATE = 
-    "<tr><td>{RANK}</td><td>{REAL NAME}</td><td>{PSEUDONYMS}</td><td>{ADDRESS}</td><td>{COLLEGE}</td><td>{WATER_STATUS}</td><td>{CRIME}</td><td>{REDEMPTION}</td><td>{NOTES}</td></tr>"
-        """
-        player_rows = []
-        police_rows = []
-        dead_player_rows = []
-        for playerID in wanted_players:
-            player = ASSASSINS_DATABASE.get(playerID)
-            (_, crime, redemption) = wanted_players[playerID]
-            player_rows.append(
-                PLAYER_TABLE_ROW_TEMPLATE.format(
-                    REAL_NAME=escape(player.real_name),
-                    PSEUDONYMS=escape(player.all_pseudonyms()),
-                    ADDRESS=escape(player.address),
-                    COLLEGE=escape(player.college),
-                    WATER_STATUS=escape(player.water_status),
-                    CRIME=escape(crime),
-                    REDEMPTION=escape(redemption),
-                    NOTES=escape(player.notes)
+            wanted_manger.add_event(e)
+            death_manager.add_event(e)
+
+        current_wanted_players = wanted_manger.get_wanted_players(now)
+        wanted_players = [player for player in current_wanted_players if not player.is_police and not death_manager.is_dead(player)]
+        wanted_police = [player for player in current_wanted_players if player.is_police and not death_manager.is_dead(player)]
+        # TODO Put dead police on the corrupt list if they have been made re-corrupt after dying, I think
+        dead_players_times = []
+        dead_police_times = []
+        # Support for police dying multiple times
+        # Intentionally has multiple copies of police if they die while corrupt multiple times
+        for id in set(death_manager.get_dead()):

```

Comment:
You shouldn't use `id` as the variable name here. It shadows the built-in function `id`, and redefining that (even if just in a local scope) is regarded as bad practice.